pkgs/applications/virtualization/podman/default.nix

```diff
@@ -16,13 +16,13 @@
 
 buildGoModule rec {
   pname = "podman";
-  version = "3.0.0";
+  version = "3.0.1";
 
   src = fetchFromGitHub {
     owner = "containers";
     repo = "podman";
     rev = "v${version}";
-    sha256 = "1dsriw2vjzjaddxdhl3wbj2ppnsyi29f4bjwc8lzyz20wfwx4ay4";
+    sha256 = "sha256-+z28Z0KvaJa32+eTGFsNX8g/WVd3BohKoBnNPU/kpWM=";
```

Comment:
```suggestion
    hash = "sha256-+z28Z0KvaJa32+eTGFsNX8g/WVd3BohKoBnNPU/kpWM=";
```

Reply:
I though we we're waiting for something to happen before doing `sha256` -> `hash`?

Reply:
We already have >1400 SRI hashes in nixpkgs.

Reply:
There are about 1100 occurrences of `sha256 = "sha256-` in nixpkgs, and I do seem to recall something similar to what @zowoq is saying.

I suggest a treewide PR when the time is right.

Reply:
I was thinking of updating the nixpkgs-update bot code to make it start fixing this and sending a treewide PR.  Curious if there's a status update for the state of SRI hashes on discourse / github issue somewhere?  Obviously can wait if there's a reason, but it feels really ugly to have n+1 ways of doing things in the meantime!